exist-parent/pom.xml

```diff
@@ -1232,10 +1254,15 @@
 
     <distributionManagement>
         <snapshotRepository>
-            <id>exist-db-snapshots</id>
-            <name>Evolved Binary - eXist-db Snapshots</name>
-            <url>https://repo.evolvedbinary.com/repository/exist-db-snapshots/</url>
+            <id>github</id>
+            <name>GitHub Registry for Snapshots</name>
```

Comment:
Is it valid to set the `snapshotRepository` by just the **id**? Would be worth to try.